commands/management/add_mysql.go

```diff
@@ -115,6 +117,44 @@ func (cmd *addMySQLCommand) GetAddress() string {
 	return cmd.Address
 }
 
+func (cmd *addMySQLCommand) processGlobalAddFlags() (serviceName string, socket string, host string, port uint16, err error) {
+	serviceName = cmd.GetServiceName()
+	if *addServiceNameFlag != "" {
+		serviceName = *addServiceNameFlag
+	}
+
+	socket = cmd.Socket
+	address := cmd.GetAddress()
+	var portI int
+	if socket != "" && (address != "" || *addHostFlag != "" || *addPortFlag != 0) {
+		return "", "", "", 0, fmt.Errorf("both socket and address are passed")
+	} else if socket == "" {
+		if address == "" {
+			address = "127.0.0.1:3306"
+		}
+		var portS string
+		host, portS, err = net.SplitHostPort(address)
```

Comment:
We need to add a check if the port is in the valid range, otherwise, we hide the error.

Reply:
We don't check it anywhere :) Good point